### PR TITLE
SkipFlakey: TestIntegration_MercuryV1 + TestIntegration_MercuryV2_Plugin

### PR DESCRIPTION
**Pull request by DeividasK**
Ticket: https://smartcontract-it.atlassian.net/browse/MERC-5697

### CODE REVIEW

**Review by cedric-cordenier — DISMISSED** [review]

**Comment by jmank88** [comment]
Do you have a link to an instance of this failure?

**Comment by DeividasK** [comment]
> Do you have a link to an instance of this failure?

https://github.com/smartcontractkit/chainlink/actions/runs/9551586686/job/26326258179?pr=13580

File: core/services/ocr2/plugins/mercury/integration_plugin_test.go [hunk 1]
```diff
@@ -6,17 +6,19 @@ import (
 	"testing"
 
 	"github.com/smartcontractkit/chainlink/v2/core/config/env"
+	"github.com/smartcontractkit/chainlink/v2/core/internal/testutils"
 )
 
 func TestIntegration_MercuryV1_Plugin(t *testing.T) {
+	testutils.SkipFlakey(t, "https://smartcontract-it.atlassian.net/browse/MERC-5697")
```
Comment (jmank88) [inline core/services/ocr2/plugins/mercury/integration_plugin_test.go]: The link in the ticket is for `TestIntegration_MercuryV1`

**Review by jmank88 — CHANGES_REQUESTED** [review]

File: core/services/ocr2/plugins/mercury/integration_plugin_test.go (hunk 1 above)
Comment (DeividasK) [reply core/services/ocr2/plugins/mercury/integration_plugin_test.go]: Could the owner of the ticket create subtasks? I assume that all flakes will be handled in the same sprint/context.

Comment (jmank88) [reply core/services/ocr2/plugins/mercury/integration_plugin_test.go]: I mean that you have marked the wrong test to be skipped. Do you have a link to an instance of `TestIntegration_MercuryV1_Plugin` failing? The existing link is for `TestIntegration_MercuryV1`

Comment (DeividasK) [reply core/services/ocr2/plugins/mercury/integration_plugin_test.go]: You are absolutely right 🙈 updated

**Review by jmank88 — APPROVED** [review]

**Review by cedric-cordenier — APPROVED** [review]

**Pull request merged**